README.md

```diff
@@ -27,12 +27,57 @@ The standardized MAFs generated by `standardize_mutation_data.py` will be writte
 
 **Sequence Source** ( `-s | --sequence-source` ) : The sequence source is a required field for a standard MAF as defined by the TCGA project. Accepted values are either WGS and WXS. For GENIE, WXS is used since GENIE samples are sequenced with targeted gene panels.
 
-**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts are located:
+**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts and data files are located:
 
-- merge_mafs.py
-- standardize_mutation_data.py
-- annotator.jar
+- merge_mafs.py (located in this repository)
+- standardize_mutation_data.py (located in this repository)
+- annotator.jar - The genome nexus annotation pipeline (See how to build the jar file [here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline))
+- application.properties (optional) - custom properties for the annotation pipeline. The common use case is to specify custom genome nexus url with `genomenexus.base` property.
+- AwsSsl.truststore - custom trust store. Used to hold an ssl certificate (aka HTTPS certificate) of trusted websites. In our case the trust store holds certificates of custom genome nexus instance. See how to create trust store and upload the certificate [here](#create-a-trust-store).
```

Comment:
I think this needs a little more explanation. For one thing, it reads like a requirement ... especially coming after the previous line which is marked as (optional). If users of annotation-tools are ok with using the default genome-nexus server .. or any genome-nexus deployment which has adequate/valid SSL certificates installed then I don't think this step would be necessary. This case would only be required of users who wish to use annotations from a genome-nexus deployment which had invalid/self-signed SSL certificates or one which did not offer https service at all. And the primary user of annotation-tools (AACR Genie) doesn't have this need as far as I know. So I think this guidance is kind of extraneous for them. I guess I would recommend creating a separate docs page explaining the use case of having a genome-nexus with a self-signed certificate, and then presenting these steps for how to handle it. On the main README.md page we could put a link to that page with a short description like "For guidance when using your own custom genome-nexus deployment, see &lt;link>"

Reply:
@sheridancbio @inodb @ao508 

The `annotation_suite_wrapper.sh` uses `uniprot` isoforms atm. Shouldn't it be `mskcc` instead?

```
GENOME_NEXUS_ANNOTATOR_ISOFORM="uniprot"
```

Reply:
yes, we should switch the default to be `mskcc`, but GENIE is still using `uniprot` unf

Reply:
@ruslan-forostianov We do not run annotation-tools locally (as far as I know). This code was written to suit the needs of AACR GENIE.

I see what you mean about the requirement for having the trust store file available when running the script. I must have remembered this incorrectly. It must mean that we provided the AwsSsl.truststore files to AACR GENIE for their use. Since it is actually a requirement then I think your original documentation statements were correct.

Reply:
Yes - the AwsSsl.truststore file was provided to us for accessing the genome nexus instance that was implemented for the GENIE usecase.